EventListener/RouteAnnotationEventListener.php

```diff
@@ -195,12 +217,30 @@ public function getOptions($name, Route $route)
     protected function getUrlConcrete($name, $options)
     {
         try {
-            return new UrlConcrete(
-                $this->getRouteUri($name),
+            $params = [];
```

Comment:
The method is called `getUrlConcrete`, maybe we can stick to that for now, but decorate that url later ?

Reply:
You right, maybe is better introduce a new method to handle this case